src/toolbox/utils/hex-to-int.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { hexToInt } from "./hex-to-int";
+import { expect } from "chai";
+import "mocha";
+
+describe("hexToInt", () => {
+  it("should correctly return a large value", () => {
```

Comment:
We probably want to test values that aren't six digits, so from a range of hex values.
We want to be able to quickly scale these tests up and down as we have more cases, so what if we setup an array of input and expected results, and then loop through them to test. You can reference the are-equal.spec.ts file for an example.